main.py

```diff
@@ -44,82 +44,77 @@ def evaluateTestQuestions(test_choice):
+                elif test_choice == 1:
+                    for i in range(len(line)):
+                        if i >= 2 and line[i] != "":
+                            numberOfCorrectAnswers += 1
+                
+                # Getting the correct answers
+                correctAnswers = []
+                for i in range(numberOfCorrectAnswers):
+                    answer = line[2 + i].lower().strip()
+                    correctAnswers.append(answer)
                 correctAnswers.sort()
-                for i in range(len(answer)):
+                
+                # Comparing the system and correct answers
+                correct = True
+                if numberOfSystemAnswers != numberOfCorrectAnswers:
+                    correct = False
                     if local_verbose:
-                        print(systemAnswers[i] + " vs " + correctAnswers[i])
-                    if systemAnswers[i] != correctAnswers[i]:
-                        correct = False
-                # Adding the correctness to the totals
+                        if numberOfSystemAnswers == 0:
+                            print("No answer could be given by the system.")
+                        else:
+                            print("The given answer(s) was not expected.")
+                elif numberOfSystemAnswers > 0:
+                    if local_verbose:
+                        print("Given answers vs correct answers:")
+                    for i in range(numberOfSystemAnswers):
+                        if local_verbose:
+                            print(systemAnswers[i] + " vs " + correctAnswers[i])
+                        if systemAnswers[i] != correctAnswers[i]:
+                            correct = False
+                elif numberOfSystemAnswers == 0 and local_verbose:
+                     print("There is no correct answer.")
+
+                # Adding the result to the totals
                 if correct:
+                    if local_verbose:
+                        print("Correct")
                     total_correct += 1
                 else:
+                    if local_verbose:
+                        print("Incorrect")
                     total_incorrect += 1
-            # This is only 1 correct answer
-            elif len(answer) == 1:
-                systemAnswer = answer[0].lower()
-                correctAnswer = line[2].lower().strip()
-                if local_verbose:
-                    print(systemAnswer +  " vs " + correctAnswer)
-                if systemAnswer == correctAnswer:
-                    total_correct += 1
-                else:
-                    total_incorrect += 1
-            # There aren't any correct answers
-            elif len(answer) == 0:
+                
+                # Printing the current results 
+                print("\rCorrect answers: " + str(total_correct) + "/" + str(total_lines) + ", Wrong answers: " + str(total_incorrect) + "/" + str(total_lines), end="")
```

Comment:
A bit of a small concern, but could you split this statement over multiple lines? (Like I have done previously?)

Reply:
Done